stablehlo/dialect/TypeInference.h

```diff
@@ -19,8 +19,11 @@ limitations under the License.
 
 #include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/Dialect.h"
 #include "mlir/IR/Location.h"
+#include "mlir/IR/Region.h"
 #include "mlir/IR/Types.h"
+#include "mlir/IR/Value.h"
```

Comment:
Were Region and Value missing from this file? Changes are only for `Dialect` by the looks of it.

Reply:
Good point! I've reverted the change.